src/Applications/LBAF.py

```diff
@@ -379,31 +382,34 @@ def get_output_file_stem(params, n_ranks):
         "initial sent volumes",
         logger=lgr)
 
-    # Prepare input data for rank order enumerator
```

Comment:
Please add `info` level log comment here, saying that brute force optimization will **not** take place

Reply:
@ppebay done